systemd/oguri.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=A very nice animated wallpaper daemon for Wayland compositors
```

Comment:
Can you remove the value judgment "very nice" ? (I know it's from upstream but still)

Reply:
lol, sure

Reply:
Done